### ISSUE TITLE
Textarea -  Default story in the Test folder has accessibility violation "Form elements must have labels"

### ISSUE THREAD

**Issue by ZhuoyuJin** [Sage/carbon#5086]
### Current behaviour

Repro:
1. Go to Textarea component Test folder, default story in Storybook https://carbon.sbc-ui.com/?path=/story/textarea-test--default
2. For Accessibility, there's a violation "Form elements must have labels"
![Image 25-04-2022 at 09 58](https://user-images.githubusercontent.com/72838192/165058837-264d9ce7-9d44-4641-88e0-505a1ffa596e.jpg)

### Expected behaviour

No accessibility violations occur. 

### CodeSandbox or Storybook URL

https://carbon.sbc-ui.com/?path=/story/textarea-test--default

### Carbon Version

latest

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by edleeks87** [comment]
FE-5144
